lapce-proxy/src/dispatch.rs

```diff
@@ -1265,3 +1227,76 @@ fn git_get_remote_file_url(workspace_path: &Path, file: &Path) -> Result<String>
         file.strip_prefix(workspace_path)?.to_str().unwrap()
     ))
 }
+
+fn start_global_search_job(
+    proxy_rpc: ProxyRpcHandler,
+    global_search_rx: Receiver<()>,
+) {
+    loop {
```

Comment:
Can someone explain why this loop is necessary here? It doesn't work if you remove it. Yet people keep telling that it should work without it. The documentation says that the channel will yield `None` if the sender is empty AND disconnected. But our sender here lives as long as `Dispatcher` is so I don't see why it should die.

Reply:
I am stupid :brain: that is supposed to be a `continue` not a `break`.